client/state/ui/test/selectors.js

```diff
@@ -111,6 +112,35 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getSection()', () => {
+		it( 'should return an empty object if no section is assigned', () => {
+			const section = getSection( {
+				ui: {
+					section: false
+				}
+			} );
+
+			expect( section ).to.eql( {} );
```

Comment:
If most places are reading the prop directly, this means it may have been depending on the false value.